### PR TITLE
Use new permissions system to enforce course editability

### PR DESCRIPTION
**Pull request by jrjohnson**
Depending on the users permissions a lot of data must sometimes be
loaded so this work has to be done in the route before rendering
otherwise users may see an uneditable screen followed by a pop-in of new
actions and colors.

Requires https://github.com/ilios/common/pull/199

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview for *ilios-frontend* ready!

Built with commit ace51048dad4d60f1a78a21d2d7d3882a89c7b38

https://deploy-preview-3677--ilios-frontend.netlify.com

**Comment by jrjohnson** [comment]
Needs to wait for https://github.com/ilios/common/pull/202

**Comment by jrjohnson** [comment]
Ready for review again, issues with session learning materials are documented in #3688

File: app/components/mesh-manager.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export default Component.extend({
   layout: layout,
   classNames: ['mesh-manager'],
   terms: null,
+  editable: true,
```
Comment (stopfstedt) [inline app/components/mesh-manager.js]: should be `false` by the default, like the others.

**Pull request merged**